test_rooms/Test1.json

```diff
@@ -0,0 +1 @@
n":{"x":0,"y":19},"size":{"x":30,"y":10,"z":31}},"name":"Livingroom"},{"element":{"@type":"cuboid","position":{"x":4,"y":1,"z":19},"size":{"x":4,"y":8,"z":1},"material":"AIR"},"name":"LivingToBedroomDoor"},{"element":{"@type":"cuboid","position":{"x":29,"y":1,"z":24},"size":{"x":1,"y":8,"z":4},"material":"AIR"},"name":"OutsideDoor"},{"element":{"@type":"cuboid","position":{"x":10,"y":1,"z":44},"size":{"x":12,"y":2,"z":4},"material":"WOOD"},"name":"BigCoachBase"},{"element":{"@type":"cuboid","position":{"x":10,"y":3,"z":48},"size":{"x":12,"y":2,"z":1},"material":"WOOD"},"name":"BigCoachBack"},{"element":{"@type":"cuboid","position":{"x":10,"y":1,"z":30},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"ulTableLeg"},{"element":{"@type":"cuboid","position":{"x":20,"y":1,"z":30},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"urTableLeg"},{"element":{"@type":"cuboid","position":{"x":10,"y":1,"z":35},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"llTableLeg"},{"element":{"@type":"cuboid","position":{"x":20,"y":1,"z":35},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"lrTableLeg"},{"element":{"@type":"cuboid","position":{"x":10,"y":3,"z":30},"size":{"x":11,"y":1,"z":6},"material":"WOOD"},"name":"mainTable"},{"element":{"@type":"cuboid","position":{"x":15,"y":1,"z":5},"size":{"x":11,"y":2,"z":8},"material":"WOOD"},"name":"BedBase"},{"element":{"@type":"cuboid","position":{"x":25,"y":3,"z":5},"size":{"x":1,"y":1,"z":8},"material":"WOOD"},"name":"BedBack"},{"element":{"@type":"cuboid","position":{"x":3,"y":1,"z":3},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"ulChairLeg"},{"element":{"@type":"cuboid","position":{"x":5,"y":1,"z":3},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"urChairLeg"},{"element":{"@type":"cuboid","position":{"x":3,"y":1,"z":5},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"llChairLeg"},{"element":{"@type":"cuboid","position":{"x":5,"y":1,"z":5},"size":{"x":1,"y":2,"z":1},"material":"WOOD"},"name":"lrChairLeg"},{"element":{"@type":"cuboid","position":{"x":3,"y":3,"z":3},"size":{"x":3,"y":1,"z":3},"material":"WOOD"},"name":"ChairBase"},{"element":{"@type":"cuboid","position":{"x":3,"y":4,"z":2},"size":{"x":3,"y":2,"z":1},"material":"WOOD"},"name":"ChairBack"},{"element":{"@type":"cuboid","position":{"x":2,"y":1,"z":6},"size":{"x":1,"y":4,"z":1},"material":"WOOD"},"name":"uDeskLeg"},{"element":{"@type":"cuboid","position":{"x":6,"y":1,"z":6},"size":{"x":1,"y":4,"z":1},"material":"WOOD"},"name":"urDeskLeg"},{"element":{"@type":"cuboid","position":{"x":2,"y":1,"z":9},"size":{"x":1,"y":4,"z":1},"material":"WOOD"},"name":"llDeskLeg"},{"element":{"@type":"cuboid","position":{"x":6,"y":1,"z":9},"size":{"x":1,"y":4,"z":1},"material":"WOOD"},"name":"lrDeskLeg"},{"element":{"@type":"cuboid","position":{"x":2,"y":4,"z":6},"size":{"x":5,"y":1,"z":4},"material":"WOOD"},"name":"DeskBase"},{"element":{"@type":"ignition","position":{"x":14,"y":3,"z":32},"size":{"x":2,"y":1,"z":2}},"name":"TableIgnition"}]}
```

Comment:
How about we put this file in /rooms and add it as an exception in .gitignore?
e.g. `!/rooms/Test1.json`

Reply:
Thats good idea